docs/schema.md

```diff
@@ -319,6 +319,17 @@ The defined database prefixes and their "home" databases are:
         </ul>
       </td>
     </tr>
+    <tr>
+      <td><code>RSEC</code></td>
+      <td><a href="https://github.com/RConsortium/r-advisory-database">RConsortium Advisory Database</a></td>
+      <td>
+        <ul>
+          <li>How to contribute: <a href="https://github.com/RConsortium/r-advisory-database#readme">https://github.com/RConsortium/r-advisory-database#readme</a></li>
+          <li>Source URL: <code>https://osv.dev/vulnerability/&lt;ID&gt;</code></li>
+          <li>OSV Formatted URL: <code>https://github.com/RConsortium/r-advisory-database/blob/main/&lt;package&gt;/&lt;ID&gt;.yaml</code> (unofficial)</li>
```

Comment:
@oliverchang  I think this is still wrong, missing `/vulns` in the path :D

Reply:
Doh, thanks for catching.

Can you please raise a PR to fix this?

Reply:
Doh, thanks for catching.

Can you please raise a PR to fix this?

Reply:
Sure, I'll push it when I get a minute.